cmd/dashboard.go

```diff
@@ -160,8 +163,8 @@ dapr dashboard -k -p 9999
 				portForward.Stop()
 			}()
 
-			// url for dashboard after port forwarding.
-			var webURL string = fmt.Sprintf("http://%s:%d", dashboardHost, dashboardLocalPort)
+			// url for dashboard after port forwarding
```

Comment:
The linter is failing because this comment doesn't have a dot `.` at the end.

Reply:
Fix it.